blogapplication

```diff
@@ -0,0 +1,50 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
why are we adding a private key to the repo ?

Reply:
Ideally we should never commit private keys in git. If we need to use any private key we should use environment variables. Let me know if you dont know how to use them.